dns/pfSense-pkg-bind9/files/usr/local/pkg/bind_zones.xml

```diff
@@ -228,8 +234,8 @@
 			<fieldname>temp03</fieldname>
 		</field>
 		<field>
-			<fielddescr>TLL</fielddescr>
-			<fieldname>tll</fieldname>
+			<fielddescr>TTL</fielddescr>
+			<fieldname>ttl</fieldname>
```

Comment:
Changing the field should also be accompanied by some code to carry over the user's old setting for the field. Otherwise, the user's existing value will be lost.

The TLL/TTL correction doesn't appear to be related to the RPZ changes in this PR, is there some reason they need to happen together? If not, the TLL/TTL change should be its own PR so the two sets of changes are distinct.